V1_EngagingWithUs.md

```diff
@@ -0,0 +1,49 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: engaging-with-us
+is: 'published'
+---
+
+## Engaging with us
+
+We are continually adding value and want to include you as early as possible in our development process so we can include 
+your input and feedback. Some features are stable (in that they will not change), others are in progress of being built which we call **beta** 
+and others are things we want to try and discover what would work best, we call **experimental**. Additionally, when we are about to 
+introduce something new and want to get feedback on the concepts and understanding before we expose API’s, so that your 
+feedback can inform and guide the future API, we will call these **future**. Also, **future** may, (but not always), include the ability for you to 
+play with an API, through one of our tools (like the GraphQL Explorer), so you can have some experience with it and give us 
+feedback to help inform and improve things.
+
+API’s that are going to be going out of service will be called **deprecated**, which means that while this is still available 
+for developers to use, it will be removed and calls to this API will no longer work. They will be kept on to provide backward 
+compatibility, and to provide time for developers to update affected code.
+
+You will see these various categories called out on those specific APIs. All other, undecorated API references are 
+considered published and are fully supported. 
+
+To provide more transparency in how we’re working and what we’re building, we will surface our content in markdown files in 
+the github repo - [github.com/watsonwork/watsonwork-developer-docs](https://github.com/watsonwork/watsonwork-developer-docs) so that you can engage with us through github to inform 
+and provide feedback to help make and design great APIs.
+
```

Comment:
Include here a paragraph on how and where to provide feedback?

- something wrong / incorrect / confusing with doc and/or with the results experienced with an API
- a reaction or comment to a experimental, beta, future API proposal / doc
- a suggestion for something to be included / modified
- other ?

Reply:
also include that in addition to above feedback we welcome Pull Requests with suggested changes, tutorials, how-tos, example references, etc.

Reply:
ok all set